src/main/resources/config/application.yml

```diff
@@ -276,6 +276,9 @@ info:
     display-ribbon-on-profiles: "dev"
     sentry:
         dsn: https://8c6b41ec2d4245e8bd3ec9541d53f625@sentry.io/1440029 # Leave empty to disable Sentry, must be a valid URI
+    postHog: # Leave empty to disable PostHog
```

Comment:
In order for this to be consistent with the rest of the file it should be named `post-hog`